frontend_tests/node_tests/bot_data.js

```diff
@@ -115,37 +112,36 @@ assert.equal(bot_data.get(314).full_name, 'Outgoing webhook');
     }());
 
     (function test_owner_can_admin() {
-        var bot;
+        let bot;
 
-        bot_data.add(_.extend({owner: 'owner@zulip.com'}, test_bot));
+        bot_data.add({owner: 'owner@zulip.com', ...test_bot });
 
         bot = bot_data.get(43);
         assert(bot.can_admin);
 
-        bot_data.add(_.extend({owner: 'notowner@zulip.com'}, test_bot));
+        bot_data.add({owner: 'notowner@zulip.com', ...test_bot });
```

Comment:
It's weird to have a space after `test_bot` but not a space before `owner`.

Reply:
Yep, i will fix this.